.github/workflows/ci.yml

```diff
@@ -107,13 +107,21 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        os: [Ubuntu, MacOS]
+        # Run on both x86 and Arm MacOS according to:
+        # https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories
+        os: [ubuntu-latest, macos-12, macos-14]
```

Comment:
Hmm... macos-latest should already be running the jobs on M1, no?

Reply:
Ah, this is unexpected, they appear to have started a 12 week roll out of `macos-latest` to be M1 shortly after my PR: https://github.blog/changelog/2024-04-01-macos-14-sonoma-is-generally-available-and-the-latest-macos-runner-image/

I checked a recent job and pip is not yet affected by this and is still on macos-12 (https://github.com/pypa/pip/actions/runs/8601739554/job/23569705793):

> Operating System
>   macOS
>   12.7.4
>   21H1123


It's interesting they did this because I found a lot of community feedback the new images break lots of workflows, but these seemed to mostly stem from limitations of the M1 chips so perhaps GitHub just wanted to finally bit the bullet.

I only found one issue with pip's tests on the M1 runner, and that isn't SVN isn't installed by default. I will make a seperate PR for that now, but I will leave this PR open as to whether to explicitly support *both* x86 and ARM or whether to just support what GitHub provides as latest.